File: algorithms/search/linearSearch/f_linearSearch.java
import java.time.Instant;
import java.time.Duration;

public class f_linearSearch {
  public static void main(String[] args) {
    int[] unsortedNumbers = { 2, 3, 1, 10, 7, 8 };
    int target = 8;
    boolean found = false;

    Instant clockStart = Instant.now();
    for (int i = 0; i < unsortedNumbers.length; i++) {
      if (unsortedNumbers[i] == target) {
        found = true;
      }
    }
    Instant clockEnd = Instant.now();

    System.out.println(found);
    System.out.println("Elapsed time: " + Duration.between(clockEnd, clockStart).toString());
  }
}

File: algorithms/search/linearSearch/f_linearSearch.kt
import java.time.Duration
import java.time.Instant

fun main() {
    val unsortedNumbers = intArrayOf(2, 3, 1, 10, 7, 8)
    val target = 2
    var found = false

    val clockStart = Instant.now();
    for (number in unsortedNumbers) {
        if (number == target) { found = true }
    }
    val clockEnd = Instant.now()

    println(found)
    println("Elapsed time: " + Duration.between(clockEnd, clockStart).toString())
}